pkg/knuu/instance_helper.go

```diff
@@ -334,44 +334,34 @@ func (i *Instance) cloneWithSuffix(suffix string) *Instance {
 	}
 
 	return &Instance{
-		name:                     i.name + suffix,
-		k8sName:                  i.k8sName + suffix,
-		imageName:                i.imageName,
-		state:                    i.state,
-		instanceType:             i.instanceType,
-		kubernetesService:        i.kubernetesService,
-		builderFactory:           i.builderFactory,
-		kubernetesStatefulSet:    i.kubernetesStatefulSet,
-		portsTCP:                 i.portsTCP,
-		portsUDP:                 i.portsUDP,
-		command:                  i.command,
-		args:                     i.args,
-		env:                      i.env,
-		volumes:                  i.volumes,
-		memoryRequest:            i.memoryRequest,
-		memoryLimit:              i.memoryLimit,
-		cpuRequest:               i.cpuRequest,
-		policyRules:              i.policyRules,
-		livenessProbe:            i.livenessProbe,
-		readinessProbe:           i.readinessProbe,
-		startupProbe:             i.startupProbe,
-		isSidecar:                false,
-		parentInstance:           nil,
-		sidecars:                 clonedSidecars,
-		files:                    i.files,
-		ingress:                  i.ingress,
-		externalDns:              i.externalDns,
-		otlpPort:                 i.otlpPort,
-		prometheusPort:           i.prometheusPort,
-		prometheusJobName:        i.prometheusJobName,
-		prometheusScrapeInterval: i.prometheusScrapeInterval,
-		jaegerGrpcPort:           i.jaegerGrpcPort,
-		jaegerThriftCompactPort:  i.jaegerThriftCompactPort,
-		jaegerThriftHttpPort:     i.jaegerThriftHttpPort,
-		otlpEndpoint:             i.otlpEndpoint,
-		otlpUsername:             i.otlpUsername,
-		otlpPassword:             i.otlpPassword,
-		jaegerEndpoint:           i.jaegerEndpoint,
+		name:                  i.name + suffix,
+		k8sName:               i.k8sName + suffix,
```

Comment:
[proposal]: I would pass it for now, but we need a checker(issue to be created) that stops creating an instance if the suffix has wrong conventions. E.g. if the user passes a space symbol with a proper char set?

Reply:
https://github.com/celestiaorg/knuu/issues/15